app/models/deploy_service.rb

```diff
@@ -108,4 +109,10 @@ def update_github_deployment_status(deploy)
       GithubDeployment.new(deploy).update_github_deployment_status(@deployment)
     end
   end
+
+  def send_sse_deploy_update(type, deploy)
+    return unless deploy.persisted?
+    rebased_deploy = JSON.parse(DeploySerializer.new(deploy).to_json)["deploy"]
```

Comment:
Thanks, after digging into the serializer a bit I figured it out. I'm still not sure how it magically works on the controller side without instantiation though.